client/src/components/controls/ColorPicker.tsx

```diff
@@ -84,7 +84,13 @@ const ColorPicker: React.FC<ColorPickerProps> = ({
             label="Hex"
             variant="outlined"
             value={textFieldValue}
-            onChange={(e) => setColor(e.target.value)}
+            onChange={(e) => {
+              let newColor = e.target.value
```

Comment:
nitpick: change it to a const

```suggestion
              const newColor = e.target.value
```